cmd/promtool/unittest.go

```diff
@@ -160,14 +160,14 @@ func (tg *testGroup) test(mint, maxt time.Time, evalInterval time.Duration, grou
 	// All this preparation is so that we can test alerts as we evaluate the rules.
 	// This avoids storing them in memory, as the number of evals might be high.
 
-	// All the `eval_time` for which we have unit tests.
-	var alertEvalTimes []time.Duration
+	// All the `eval_time` for which we have unit tests for alerts.
+	alertEvalTimesMap := make(map[time.Duration]struct{})
```

Comment:
You don't need the make here

Reply:
It panics if I don't have that. Without `make` its a `nil` map.

Reply:
`map[time.Duration]struct{}{}` should work.

Reply:
I changed it.